File: Assignment4.js
function printNumber(n) {
    console.log("number is ".concat(n));
    return;
}
printNumber(2);
var charge = 0, total = 0;
function totalCharge(dis) {
    if (dis < 2) {
        charge = 50;
        total = total + charge;
    }
    else if (dis >= 2 && dis < 5) {
        charge = 20;
        total = total + charge * dis;
    }
    else if (dis >= 5 && dis < 15) {
        charge = 18;
        total = total + charge * dis;
    }
    else if (dis >= 15) {
        charge = 15;
        total = total + charge * dis;
    }
    console.log("Total charge is ".concat(total));
}
totalCharge(8);

File: Assignment4/Assignment4.py

dis = int(input("Enter your distance: "))
total=0
charge=0
if dis<2:
   charge=50
   total=total+charge
elif dis>=2 and dis<5:
   charge=20;	
   total=total+charge*dis
elif dis>=5 and dis<15:
   charge=18;	
   total=total+charge*dis
else:
   charge=15;	
   total=total+charge*dis
    
print ("Total charge:",total)
